example/lib/main.dart

```diff
@@ -178,6 +189,22 @@ class PhoneFormFieldScreenState extends State<PhoneFormFieldScreen> {
                           setState(() => isCountryChipPersistent = v),
                       title: const Text('Persistent country chip'),
                     ),
+                    SwitchListTile(
+                      value: showDialCode,
+                      onChanged: (v) => setState(() => showDialCode = v),
+                      title: const Text('Show Dial Code'),
+                    ),
+                    SwitchListTile(
+                      value: isCountrySelectionEnabled,
+                      onChanged: (v) =>
+                          setState(() => isCountrySelectionEnabled = v),
+                      title: const Text('Is Country Selection enabled'),
+                    ),
+                    SwitchListTile(
```

Comment:
This one can be removed as per comment above about showFlagInput.

Reply:
I think in example 
We need add all attributes and user test all things

Reply:
isCountrySelectionEnabled use when i need disable change flag ( country ) but need flag display and show flag input to hide flag or show